types/did_doc.go

```diff
@@ -19,22 +19,21 @@ type DidDoc struct {
 }
 
 type VerificationMethod struct {
-	Context            []string          `json:"@context,omitempty"`
-	Id                 string            `json:"id,omitempty"`
-	Type               string            `json:"type,omitempty"`
-	Controller         string            `json:"controller,omitempty"`
-	PublicKeyJwk       map[string]string `json:"publicKeyJwk,omitempty"`
-	PublicKeyMultibase string            `json:"publicKeyMultibase,omitempty"`
+	Context              []string `json:"@context,omitempty"`
+	Id                   string   `json:"id,omitempty"`
+	Type                 string   `json:"type,omitempty"`
+	Controller           string   `json:"controller,omitempty"`
+	VerificationMaterial string   `json:"verificationMaterial,omitempty"`
```

Comment:
Here we need additional logic. `VerificationMaterial` contains json. All fields from this json should be added to the verification method.

Reply:
@askolesov Do you mean the resolved result should have the fields inside parsed verificationMaterial, This struct will remain the same right?

Reply:
Yeah... Look, if we have this stored on ledger:
```
{
Id
Type
...
VerificationMaterial: "{fieldXXX: valueX, fieldYYY: valueY}"
}
```
We want to get this on the resolver:
```
{
Id
Type
...
fieldXXX: valueX
fieldYYY: valueY
}
```

Reply:
Do you have any ideas on implementing this ?, Its not possible to have an anonymous field with interface{} type.

Reply:
This is complete